python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/ethernet-interfaces.j2

```diff
@@ -80,34 +84,38 @@
 {%                 set mode = ethernet_interface.switchport.mode | arista.avd.default('-') %}
 {%                 if ethernet_interface.switchport.access_vlan is arista.avd.defined or
                     ethernet_interface.switchport.trunk.allowed_vlan is arista.avd.defined %}
-{%                     set vlans = [] %}
+{%                     set switchport_vlans = [] %}
 {%                     if ethernet_interface.switchport.access_vlan is arista.avd.defined %}
-{%                         do vlans.append(ethernet_interface.switchport.access_vlan) %}
+{%                         do switchport_vlans.append(ethernet_interface.switchport.access_vlan) %}
 {%                     endif %}
 {%                     if ethernet_interface.switchport.trunk.allowed_vlan is arista.avd.defined %}
-{%                         do vlans.extend(ethernet_interface.switchport.trunk.allowed_vlan | arista.avd.range_expand | map('int')) %}
+{%                         do switchport_vlans.extend(ethernet_interface.switchport.trunk.allowed_vlan | arista.avd.range_expand | map('int')) %}
+{%                     endif %}
+{%                     if switchport_vlans %}
+{%                         set switchport_vlans = switchport_vlans | arista.avd.list_compress %}
+{%                     elif ethernet_interface.switchport.trunk.allowed_vlan is arista.avd.defined("none") %}
```

Comment:
I think this would just be an `else` case, since no matter how we got an empty list of vlans, it means `"none"`

Reply:
Fixed